lib/mvos/ui/widgets/create_account_form.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:flutter/material.dart';
+import 'package:playground_app/router/router.dart';
+
+
+class CAForm extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.all(8.0),
+      child: Form(
```

Comment:
needs to be statefull widget, and you need to add form key to be able to validate the whole form